src/resources/views/crud/form_content.blade.php

```diff
@@ -35,12 +35,21 @@
     <script>
     function initializeFieldsWithJavascript(container) {
       var selector;
+
       if (container instanceof jQuery) {
         selector = container;
       } else {
         selector = $(container);
       }
-      selector.find("[data-init-function]").not("[data-initialized=true]").each(function () {
+
+      var $elements = selector.find("[data-init-function]").not("[data-initialized=true]");
+
+      // If it's not a repeatable container we exclude all the repeatable fields from the initialization
+      if (!selector.hasClass('repeatable-element')) {
+        $elements = $elements.not(".repeatable-field-input");
```

Comment:
Hello @tabacitu atm I could not think of any other solution, but maybe this is a wording problem that we can fix. 

First of all, why the change in `form_content`: there are three places in the current code where we call `initializeFieldsWithJavascript`. 

The ones we are looking for are the self `form_content` that initializes the whole form, and the `repeatable` call that would initialize the repeatable elements. 

What is needed here is a way to **NOT initialize** some fields per developer instruction. Atm there is no functionality in backpack that allows this. It's here that I say this might be a wording problem.

If we changed the classe names from: `repeatable-element` to `self-initialize-container` and `repeatable-field-input` to `self-initialize-field` we are creating a feature that other developers could use to tell backpack that the fields inside that container should not be initialized by main `form` initialization script and that would make `repeatable` use that functionality instead of being "a repeatable functionality". That's what I would call the wording problem. 

Let me know what you think. 

Pedro

Reply:
That. Is fucking. AWESOME! Excellent solution @pxpm , and excellent wording for it.

Do we need to _rename_ `repeatable-element` and `repeatable-field-input` or can we just _add this feature on top_? Aren't `repeatable-element` and `repeatable-field-input` used for other purposes, other than initialising the JS?

Reply:
`repeatable-element` is already used by repeatable functionality, we can add the `self-initialize-container` class along with. As for `repeatable-field-input` that's a class I created for this functionality, so it could just be renamed to `self-initialize-field`.

Reply:
@pxpm we decided in our conversation that we should try to move ALL subfields outside the main form, using `@push('before_scripts')` or something like that. If that happens we'll no longer have that problem.

Waiting for your alternative PR with that solution - I have high hopes for it :smile: